back/src/utils/anonym.mjs

```diff
@@ -0,0 +1,19 @@
+const generateUniqueEmail = () => {
+    const randomString = Math.random().toString(36).substring(2, 8);
+    const domain = 'anonym.com';
+    const email = `user_${randomString}@${domain}`;
+    return email;
+};
+
+export const anonymizeUserData = () => {
+    const user = {
+        firstname: "Anonyme",
+        lastname: "Anonyme",
+        email: generateUniqueEmail(),
```

Comment:
Why should we generate unique email ? do we need it for later, if not we can just put empty string or the same email for everyone

Reply:
You're right i did think that it was necessary to do that for each user but since we anonymise it we can make the same email for everyone